searchlib/src/vespa/searchlib/engine/errorcodes.h

```diff
@@ -9,26 +9,8 @@ namespace search::engine {
  * Used in error_code field in search::fs4transport::PCODE_ERROR packets.
  **/
 enum ErrorCode {
-    ECODE_NO_ERROR            = 0,
-    ECODE_GENERAL_ERROR       = 1,
-    ECODE_QUERY_PARSE_ERROR   = 2,
-    ECODE_ALL_PARTITIONS_DOWN = 3,
-    ECODE_ILLEGAL_DATASET     = 4,
-    ECODE_OVERLOADED          = 5,
-    ECODE_NOT_IMPLEMENTED     = 6,
-    ECODE_QUERY_NOT_ALLOWED   = 7,
-    ECODE_TIMEOUT             = 8
+    ECODE_QUERY_PARSE_ERROR   = 2
```

Comment:
why do we still need this when error codes and messages are completely removed from the api?

Reply:
errorcodes removed, the others will go in a separate PR.